tools/c7n_azure/c7n_azure/entry.py

```diff
@@ -27,6 +27,7 @@
 import c7n_azure.resources.network_security_group
 import c7n_azure.resources.web_app
 import c7n_azure.resources.access_control  # noqa: F401
+import c7n_azure.resources.appserviceplan
```

Comment:
The `# noqa: F401` flake8 bypass has to be on the last import line, this will fix your build

Reply:
Cool. Thanks!